packages/@jsii/go-runtime-test/project/compliance_test.go

```diff
@@ -718,14 +716,28 @@ func (suite *ComplianceSuite) TestAbstractMembersAreCorrectlyHandled() {
 }
 
 func (suite *ComplianceSuite) TestCanOverrideProtectedSetter() {
-	suite.FailTest("unable to access protected members", "https://github.com/aws/jsii/issues/2673")
 	assert := suite.Assert()
 	challenge := "Bazzzzzzzzzzzaar..."
-	overridden := myOverridableProtectedMember{calc.NewOverridableProtectedMember()}
+	overridden := newTestCanOverrideProtectedSetterOverridableProtectedMember()
```

Comment:
This test didn't implement the overrides correctly and was using a shared struct that didn't have the necessary functionality to assert this test.

Created a dedicated struct for this test and apply our standard overrides pattern.